src/java.base/share/classes/com/sun/crypto/provider/PBES2Parameters.java

```diff
@@ -239,8 +239,8 @@ protected void engineInit(byte[] encoded)
         }
         cipherAlgo = parseES(pBES2_params.data.getDerValue());
 
-        this.pbes2AlgorithmName = new StringBuilder().append("PBEWith")
-            .append(kdfAlgo).append("And").append(cipherAlgo).toString();
+        this.pbes2AlgorithmName = "PBEWith" +
+                kdfAlgo + "And" + cipherAlgo;
```

Comment:
nit: move to one line if less than 80 chars.

Reply:
75 chars. It's now one line.